modules/machine-translation/settings-preview-machine-translation.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class to advertize the Machine Translation module.
+ *
+ * @since 3.6
+ */
+class PLL_Settings_Preview_Machine_Translation extends PLL_Settings_Module {
+	/**
+	 * Stores the display order priority.
+	 *
+	 * @var int
+	 */
+	public $priority = 85;
```

Comment:
I think it's free and still < License key prio:
```suggestion
	public $priority = 90;
```

Reply:
Done in [72fb3a5](https://github.com/polylang/polylang/pull/1420/commits/72fb3a59c4b05435a120e5467ff525323cf5fe15).